README.md

```diff
@@ -10,6 +10,23 @@ You can obtain this POM from Maven Central:
 The Expedia Group OSS Parent POM provides a Maven parent POM that is intended to be used by Expedia Group projects that require certain 
 base functionality and need to publish artifacts to [Maven Central](https://search.maven.org/).
 
+# Plugins
+## [Jib Maven plugin](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin)
+This plugin builds an optimized Docker image for a Java application and uploads it to Artifactory.
+
+To setup the plugin in a child project add the following plugin references. The `git-commit-id-plugin` provides the 
+last commit time to be used for the `creationTime` and `filesModificationTime` properties in the jib plugin configuration.
+```xml
+<plugin>
+    <groupId>com.google.cloud.tools</groupId>
+    <artifactId>jib-maven-plugin</artifactId>
+</plugin>
+<plugin>
+    <groupId>pl.project13.maven</groupId>
+    <artifactId>git-commit-id-plugin</artifactId>
+</plugin>
+```
+
```

Comment:
We should also have the `Build Docker Image` section, minus the first bit in our internal version which discusses non-OSS builds, the rest is still valid for disabling the plugin and setting the timeout.